test/Infer/syn-nand.ll

```diff
@@ -0,0 +1,29 @@
+; REQUIRES: solver
+; RUN: %souper-check -infer-rhs -souper-exhaustive-synthesis -souper-exhaustive-synthesis-num-instructions=2 %solver %s > %t1
+; RUN: %FileCheck %s < %t1
+; CHECK: RHS inferred successfully
```

Comment:
please check for the actual instructions you want to see, not just for successful synthesis